packages/expo-router/src/modal/web/ModalStackRouteDrawer.web.tsx

```diff
@@ -179,6 +179,7 @@ function ModalStackRouteDrawer({
       dismissible={options.gestureEnabled ?? true}
       onAnimationEnd={handleOpenChange}
       shouldScaleBackground
+      autoFocus
```

Comment:
So this could potentially be an issue if someone has a modal but wants the background view to stay in focus?

Reply:
It's not the background view; it's links or pressables that are behind the view. As soon as we open the modal, Vaul (Radix) sets `aria-hidden` to `#root`, which is correct since screen readers should not "see" or interact with the background of the modal. The error we had was that clicking on the pressable retained "focus" on it, so using "tab" would tab through the links behind the modal. `autoFocus` moves and traps the focus into the highest mounted modal, which is the correct approach for accessibility and focus trapping. With that flag, you can use tab, and it will only jump between items inside the modal again.